deploy.sh

```diff
@@ -1,6 +1,6 @@
 set -e
 
-API="https://api.cloud.gov"
+API="https://api.fr.cloud.gov"
```

Comment:
Well, we're trying to [move tenant applications off of `*.fr.cloud.gov`](https://github.com/18F/cg-product/issues/499)... atf-eregs.18f.gov, I guess?

Related question: mind if we remove the other route bindings, so that the app only has that one?

```
$ cf routes
Getting routes for org atf-eregs / space prod as aidan.feldman@gsa.gov ...

space   host        domain          port   path   type   apps        service
prod    atf-eregs   fr.cloud.gov                         atf-eregs
prod    atf-eregs   18f.gov                              atf-eregs
prod    atf-eregs   app.cloud.gov                        atf-eregs
```